### PR TITLE
chore: remove announcement bar, redirect WCID landing page

### PR DESCRIPTION
**Pull request by ericgregory**
Removes announcement bar and redirects WCID landing page to recording now that the event is done.

### CODE REVIEW

File: docusaurus.config.ts [hunk 1]
```diff
@@ -206,10 +206,10 @@ const config = (async (): Promise<Config> => {
...
         ],
       },
-      announcementBar: {
-        id: 'innovation-day',
-        content: `📢 Join us virtually for <b>wasmCloud Innovation Day</b> on September 18! <a href="/innovation-day">Register now</a> for a day of immersive learning, demos, user talks, and more.`,
-      },
+//      announcementBar: {
+//        id: 'innovation-day',
+//        content: `📢 Join us virtually for <b>wasmCloud Innovation Day</b> on September 18! <a href="/innovation-day">Register now</a> for a day of immersive learning, demos, user talks, and more.`,
+//      },
```
Comment (vados-cosmonic) [inline docusaurus.config.ts]:
```suggestion
```

We could probably remove this right?

File: netlify.toml [hunk 2]
```diff
@@ -195,5 +195,6 @@ redirects = [
 
   { from = "/youtube", to = "https://www.youtube.com/@wasmCloud", status = 301 },
   { from = "/yt", to = "https://www.youtube.com/@wasmCloud", status = 301 },
+  { from = "/innovation-day", to = "https://www.youtube.com/watch?v=vEHbZCBmPok", status = 301 },
```
Comment (vados-cosmonic) [inline netlify.toml]:
```suggestion
  { from = "/innovation-day-2024", to = "https://www.youtube.com/watch?v=vEHbZCBmPok", status = 301 },
  { from = "/innovation-day", to = "https://www.youtube.com/watch?v=vEHbZCBmPok", status = 302 },
```

Should we add a year? We might do this again 

[EDIT] Or maybe we should have two -- one with year specified and one that will rotate the next time we do it

**Review by vados-cosmonic — COMMENTED** [review]
LGTM :rocket: 

Feel free to take or leave the comments

File: netlify.toml (hunk 2 above)
Comment (ericgregory) [reply netlify.toml]: Yeah, I like the both years approach!

File: docusaurus.config.ts (hunk 1 above)
Comment (ericgregory) [reply docusaurus.config.ts]: Think I prefer to leave it in so anyone can quickly uncomment and add an announcement bar back without looking up the syntax, but maybe we should make the contents more generic/instructive? I'll update in that direction

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**